.github/dependabot.yml

```diff
@@ -1,5 +1,14 @@
 version: 2
 updates:
+  - package-ecosystem: "cargo"
+    directory: "/src/enclave-agent"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 1
```

Comment:
Is this just so we don't get overwhelmed when turning it on for the first time?

Reply:
Mainly because it is like this in the kbs repo where I copied this from but I think it makes sense to have things updated one by one to avoid extra  dependabot rebasing